install/Dockerfile

```diff
@@ -1,36 +1,67 @@
 FROM ubuntu:23.04
 
 WORKDIR /workspace
+# Determine platform and architecture
+RUN ARCH=$(uname -m) && \
+  PLATFORM=$(uname -s | tr '[:upper:]' '[:lower:]') && \
+  if [ "$ARCH" = "aarch64" ]; then \
+  echo "AWSCLI_ARCH=aarch64" >> /etc/environment; \
+  echo "TERRAFORM_ARCH=arm64" >> /etc/environment; \
+  echo "PLATFORM_ARCH=${PLATFORM}_arm64" >> /etc/environment; \
+  elif [ "$ARCH" = "x86_64" ]; then \
+  echo "AWSCLI_ARCH=x86_64" >> /etc/environment; \
+  echo "TERRAFORM_ARCH=amd64" >> /etc/environment; \
+  echo "PLATFORM_ARCH=${PLATFORM}_amd64" >> /etc/environment; \
+  else \
+  echo "Unsupported architecture"; \
+  exit 1; \
+  fi
+
+# Source the environment file so that the variable is available in the current shell
+SHELL ["/bin/bash", "-c"]
+RUN source /etc/environment
 
 # Common
-RUN apt-get update && \
+RUN DEBIAN_FRONTEND="noninteractive" \
+  apt-get update && \
   apt-get install -y \
   gnupg \
   software-properties-common \
   unzip \
   wget \
   curl \
-  git
+  git \
+  tzdata \
+  gettext-base \
+  keyboard-configuration
```

Comment:
what is the `keyboard-configuration` package used for? I  don't care if we include it but just sparkled my curiosity.

Reply:
I actually don't think it's needed - I found an aws cli install guide that listed it but I dont actually think it's necessary. Getting rid of it and tzdata.